pkg/dock/dock.go

```diff
@@ -419,16 +464,21 @@ func (ds *dockServer) UpdateVolumeGroup(ctx context.Context, opt *pb.UpdateVolum
 
 func (ds *dockServer) DeleteVolumeGroup(ctx context.Context, opt *pb.DeleteVolumeGroupOpts) (*pb.GenericResponse, error) {
 	// Get the storage drivers and do some initializations.
-	ds.Driver = drivers.Init(opt.GetDriverName())
-	defer drivers.Clean(ds.Driver)
+	driver, err := drivers.Init(opt.GetDriverName())
```

Comment:
Why cannot be this part of the code moved to a new func, say init() and called everywhere? It is repetition of code

Reply:
I just followed the original code logic, however, yes it is repeated.
I'd rather leave this improvement to another pr.

Reply:
Ok.